snackager/src/__integration-tests__/__snapshots__/bundler.test.ts.snap

```diff
@@ -433,12 +433,24 @@ Object {
           "react-native-safe-area-context",
           "react-native-svg",
         ],
-        "size": 314823,
+        "size": 316040,
       },
     },
     "web": Object {
       "bundle.js": Object {
         "externals": Array [
+          "./arrayWithHoles.js",
+          "./arrayWithoutHoles.js",
+          "./assertThisInitialized.js",
+          "./iterableToArray.js",
+          "./iterableToArrayLimit.js",
+          "./nonIterableRest.js",
+          "./nonIterableSpread.js",
+          "./objectWithoutPropertiesLoose.js",
+          "./setPrototypeOf.js",
+          "./unsupportedIterableToArray.js",
+          "@babel/runtime/helpers/interopRequireDefault",
+          "@babel/runtime/helpers/typeof",
```

Comment:
These externals are reported for `@drafbit/ui`. This is not right, need to investigate whether this is a fluke with the externals detection or whether externals are actually new.

Reply:
Fixed by adding a custom babel plugin that rewrites incorrect `@babel/runtime/helpers/esm/...` imports in `@material-ui/core`.